imagestreams/nodejs-rhel7.json

```diff
@@ -0,0 +1,131 @@
+{
+  "kind": "ImageStream",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "nodejs",
+    "annotations": {
+      "openshift.io/display-name": "Node.js"
+    }
+  },
+  "spec": {
+    "tags": [
+      {
+        "name": "latest",
+        "annotations": {
+          "openshift.io/display-name": "Node.js (Latest)",
+          "openshift.io/provider-display-name": "Red Hat, Inc.",
+          "description": "Build and run Node.js applications on RHEL 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/s2i-nodejs-container/blob/master/8/README.md.\n\nWARNING: By selecting this tag, your application will automatically update to use the latest version of Node.js available on OpenShift, including major versions updates.",
+          "iconClass": "icon-nodejs",
+          "tags": "builder,nodejs",
+          "supports":"nodejs",
+          "sampleRepo": "https://github.com/openshift/nodejs-ex.git"
+        },
+        "from": {
+          "kind": "ImageStreamTag",
+          "name": "8"
```

Comment:
Does it make sense for `8-RHOAR` to instead be the referenced `ImageStreamTag`? I say this because I am under the impression that RHOAR RPMs tend to be more current. We will be shipping 8.11.0 early next week, for example. And we already have 10.0.0 in community, with production in pipeline. I don't know, however, if this would affect existing deployments in some way.

Reply:
It would affect anyone who's currently using the "latest" imagestreamtag: they would get switched from the existing nodejs:8 image to the rhoar nodejs:8 image.

Technically that's ok, we try to warn you that if you use the latest tag, those sort of potentially incompatible changes can happen.  (Generally the expectation is we'll move you up major versions which may or may not be compatible).

That said, i'd prefer to not make that change until we at least have some expectation that the rhoar nodejs:8 image is a direct drop in replacement for the existing scl nodejs:8 image, just to avoid breaking people unnecessarily.  If it generally has the same behavior w/ respect to processing package.json and exposing the same ports, functioning as an s2i builder image, etc, then we can probably do that.

When rhoar nodejs:10 comes out, we'd certainly switch latest to point to that.

Reply:
+1 to waiting until 10 comes out and switching latest then.

+1 to getting someone from rhoar team to confirm that nodejs:8-RHOAR is a complete/compatible replacement for nodejs:8 before we switch the 8 to refer to RHOAR distribution.

So I'd say we merge this, get it on online. Then circle back to the latest/10 changes and the switch from 8-RHOAR -> 8.